### ISSUE TITLE
Any plan to add import motion VMD support ?

### ISSUE THREAD

**Issue by Orkosara** [phyblas/mmdpaimaya#4]
Hello. I wonder if we can import VMD file ?
able to do adjustment to motion and able to add new motion in existing motion like for example add cloth physics and baked to keyframe and later export it back to VMD file would be very useful.

Also able to work with HumanIK animation retargeting tool would be very helpful too.

**Comment by phyblas** [comment]
お問い合わせありがとうございます。
VMDを読み込んでMayaで動かせるようにすることや物理のこともやりたいと思っていることです。

今のところはまだ時間がないので難しいですが、できればいつかやるつもりです。
いつになるかわかりませんが、その時まで待っていただければ幸いです。

**Comment by Orkosara** [comment]
Thank you so much, I really like your work. I will wait.

**Comment by phyblas** [comment]
これを必要としている人がいるとわかって本当に嬉しいです。とても励みになって、やる気が湧いてきました。

**Comment by Orkosara** [comment]
Yeah i agree.  I think importing animation properly will be very hard since every models are different and IK are different too..
